extensions/sql-migration/src/constants/strings.ts

```diff
@@ -229,6 +229,10 @@ export const AZURE_RECOMMENDATION_TOOLTIP_IN_PROGRESS = localize('sql.migration.
 export const AZURE_RECOMMENDATION_START = localize('sql.migration.sku.azureRecommendation.start', "Start");
 export const AZURE_RECOMMENDATION_START_POPUP = localize('sql.migration.sku.azureRecommendation.start.popup', "Starting performance data collection...");
 export const AZURE_RECOMMENDATION_STOP_POPUP = localize('sql.migration.sku.azureRecommendation.stop.popup', "Stopping performance data collection...");
+export const AZURE_RECOMMENDATION_DATA_COLLECTION_POPUP_MESSAGE_LABEL = localize('sql.migration.sku.azureRecommendation.data.collection.popup.message.label', "Where do you want to save collected data?");
```

Comment:
Generally, we show texts like, choose/select a folder/directory to collect data. Please check with PM whether they want same text.

Reply:
In current mocks it is same - https://www.figma.com/file/EC2tJwlwy4RveRYhhGiHP4/Proposal-for-UI-improvement---SQL-Migration-ADS?type=whiteboard&node-id=0-1&t=CB5YVEg0XX4fijyE-0